scripts/styleChecker.fsx

```diff
@@ -0,0 +1,266 @@
+            Console.Out.Flush()
+
+        Console.Error.WriteLine errMsg
+        raise <| ProcessFailed errMsg
+    | WarningsOrAmbiguous output ->
+        if processResult.Details.Echo = Echo.Off then
+            output.PrintToConsole()
+            Console.WriteLine()
+            Console.Out.Flush()
+
+        let fullErrMsg = sprintf "%s (with warnings?)" errMsg
+        fullErrMsg
+    |> printfn "%A"
+
+
+    // Since after installing commitlint dependencies package.json file changes, we need to
+    // run the following command to ignore package.json file
+    Process
+        .Execute(
+            {
+                Command = "git"
+                Arguments = "restore package.json"
+            },
+            Echo.Off
+        )
+        .UnwrapDefault()
+    |> ignore
+
+let StyleTypeScriptFiles() =
+    RunPrettier "--quote-props=consistent --write ./**/*.ts"
+
+let StyleYmlFiles() =
+    RunPrettier "--quote-props=consistent --write ./**/*.yml"
+
+let ContainsFiles (rootDir: DirectoryInfo) (searchPattern: string) =
+    Helpers.GetFiles rootDir searchPattern |> Seq.length > 0
+
+let GitDiff() : ProcessResult =
+
+    // Since we changed file modes in the prettier step we need the following command to
+    // make git ignore mode changes in files and doesn't include them in the git diff command.
+    Process.Execute(
+        {
+            Command = "git"
+            Arguments = "config core.fileMode false"
+        },
+        Echo.Off
+    )
+    |> ignore
+
+    let processResult =
+        Process.Execute(
+            {
+                Command = "git"
+                Arguments = "diff --exit-code"
+            },
+            Echo.Off
+        )
+
+    processResult
+
+let GitRestore() =
+    Process.Execute(
+        {
+            Command = "git"
+            Arguments = "restore ."
+        },
+        Echo.Off
+    )
+    |> ignore
+
+let InstallPrettier(version: string) =
+    Process.Execute(
+        {
+            Command = "npm"
+            Arguments = $"install prettier@{version}"
+        },
+        Echo.Off
+    )
+    |> ignore
+
+let PrintProcessResult (processResult: ProcessResult) (suggestion: string) =
+    let errMsg =
+        sprintf
+            "Error when running '%s %s'"
+            processResult.Details.Command
+            processResult.Details.Args
+
+    match processResult.Result with
+    | Success output -> output
+    | Error(_, output) ->
+        if processResult.Details.Echo = Echo.Off then
+            output.PrintToConsole()
+            Console.WriteLine()
+            Console.Out.Flush()
+
+        let fullErrMsg = errMsg + Environment.NewLine + suggestion
+        fullErrMsg
+
+    | WarningsOrAmbiguous output ->
+        if processResult.Details.Echo = Echo.Off then
+            output.PrintToConsole()
+            Console.WriteLine()
+            Console.Out.Flush()
+
+        let fullErrMsg = sprintf "%s (with warnings?)" errMsg
+        fullErrMsg
+
+    |> printfn "%A"
```

Comment:
@realmarv if an error message is being printed to the console, then Console.Error.WriteLine  should be used, not prinfn (which is equivalent to Console.WriteLine)